modules/service_container_doctrine/tests/modules/service_container_doctrine_test/service_container_doctrine_test.services.yml

```diff
@@ -0,0 +1,4 @@
+parameters:
+  annotated_plugins_auto_discovery:
+    - { owner: 'service_container_doctrine_test', type: 'Plugin1', directory: 'src/Plugin/Plugin1' }
+    - { owner: 'service_container_doctrine_test', type: 'Plugin2', directory: 'src/Plugin/Plugin2' }
```

Comment:
Use:

'sc_doctrine_test'

It should _not_ match a module name, so that it fails when there is code relying on the module name ;)